build/Targets/References.targets

```diff
@@ -120,10 +84,7 @@
     <When Condition="'$(IncludeCPSReferences)' == 'true'">
       <ItemGroup>
 
-        <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\src\Dependencies\CPS\CPS.csproj">
-          <Project>{03ab838c-1b9d-4eca-856f-476616c35f0c}</Project>
-          <Name>CPS</Name>
-        </ProjectReference>
+        <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.SDK" Version="15.0.737" PrivateAssets="None" />
```

Comment:
This is great. If we can move the version numbers to variables and have them imported as well then we would not have the need to update the target and the nuspec separately. Instead we update the variables once and the change flows everywhere.